.github/workflows/lint.yml

```diff
@@ -15,8 +15,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2.1.3
```

Comment:
this setting is required from linter@v3
https://github.com/golangci/golangci-lint-action#compatibility